protocol/app/msgs/unsupported_msgs.go

```diff
@@ -18,6 +20,41 @@ var (
 		"/cosmos.gov.v1beta1.MsgSubmitProposal":         &govbeta.MsgSubmitProposal{},
 		"/cosmos.gov.v1beta1.MsgSubmitProposalResponse": nil,
 
+		// cosmwasm
+		"/cosmwasm.wasm.v1.AcceptedMessageKeysFilter":                  nil,
+		"/cosmwasm.wasm.v1.AcceptedMessagesFilter":                     nil,
+		"/cosmwasm.wasm.v1.AllowAllMessagesFilter":                     nil,
+		"/cosmwasm.wasm.v1.ClearAdminProposal":                         nil,
+		"/cosmwasm.wasm.v1.CombinedLimit":                              nil,
+		"/cosmwasm.wasm.v1.MaxCallsLimit":                              nil,
+		"/cosmwasm.wasm.v1.MaxFundsLimit":                              nil,
+		"/cosmwasm.wasm.v1.MsgClearAdmin":                              &wasm.MsgClearAdmin{},
+		"/cosmwasm.wasm.v1.MsgClearAdminResponse":                      nil,
+		"/cosmwasm.wasm.v1.MsgIBCCloseChannel":                         &wasm.MsgIBCCloseChannel{},
+		"/cosmwasm.wasm.v1.MsgIBCSend":                                 &wasm.MsgIBCSend{},
+		"/cosmwasm.wasm.v1.MsgPinCodes":                                &wasm.MsgPinCodes{},
+		"/cosmwasm.wasm.v1.MsgPinCodesResponse":                        nil,
+		"/cosmwasm.wasm.v1.MsgRemoveCodeUploadParamsAddresses":         &wasm.MsgRemoveCodeUploadParamsAddresses{},
+		"/cosmwasm.wasm.v1.MsgRemoveCodeUploadParamsAddressesResponse": nil,
+		"/cosmwasm.wasm.v1.MsgSudoContract":                            &wasm.MsgSudoContract{},
+		"/cosmwasm.wasm.v1.MsgSudoContractResponse":                    nil,
+		"/cosmwasm.wasm.v1.MsgUnpinCodes":                              &wasm.MsgUnpinCodes{},
+		"/cosmwasm.wasm.v1.MsgUnpinCodesResponse":                      nil,
+		"/cosmwasm.wasm.v1.MsgUpdateAdmin":                             &wasm.MsgUpdateAdmin{},
```

Comment:
Allow this since it's needed by gov to update admin. It's not exclusively callable by gov, but gov can call this message when the `sender` field is `governance` module itself. https://github.com/CosmWasm/wasmd/blob/c951068274fd921f3501b0dff144eba0e6aac13b/proto/cosmwasm/wasm/v1/tx.proto#L243

Reply:
This should be handled by https://github.com/CosmWasm/wasmd/blob/d63c1bdf568ba3afc01569a4933e8e2c40f01936/x/wasm/keeper/proposal_handler_legacy.go#L52